api/client/accesslist/accesslist.go

```diff
@@ -64,9 +64,11 @@ func (c *Client) GetAccessLists(ctx context.Context) ([]*accesslist.AccessList,
 // Deprecated: Use [Client.ListAccessListsV2] instead.
 // TODO (avatus): DELETE IN 21.0.0
 func (c *Client) ListAccessLists(ctx context.Context, pageSize int, nextToken string) ([]*accesslist.AccessList, string, error) {
-	resp, err := c.grpcClient.ListAccessListsV2(ctx, &accesslistv1.ListAccessListsV2Request{
+	//nolint:staticcheck // SA1019. ListAccessLists is deprecated but will
+	// continue be supported for backward compatibility.
+	resp, err := c.grpcClient.ListAccessLists(ctx, &accesslistv1.ListAccessListsRequest{
 		PageSize:  int32(pageSize),
-		PageToken: nextToken,
+		NextToken: nextToken,
 	})
```

Comment:
Looks ok, but I think it'd be good to add a TODO with the version in which this can be updated to use the new endpoint.